hawkular-inventory-api/src/main/java/org/hawkular/inventory/api/model/DataEntity.java

```diff
@@ -141,7 +147,7 @@ static Role valueOf(String name) {
         }
 
         public Blueprint(DataRole role, StructuredData value, Map<String, Object> properties) {
-            super(properties);
+            super(role.name(), properties);
```

Comment:
this is called from the no-arg constructor (see below line#168). role will always be null, thus, NPE.